src/VisualStudio/Core/Def/Implementation/ProjectSystem/CPS/IWorkspaceProjectContext.cs

```diff
@@ -42,5 +42,7 @@ internal interface IWorkspaceProjectContext : IDisposable
 
         void StartBatch();
         void EndBatch();
+
+        void ReorderSourceFiles(IEnumerable<string> filePaths);
```

Comment:
Consider doc'ing requirements here.

Reply:
None of the methods here have docs on them. Is this intentional @jasonmalinowski ?

Reply:
Well, this is something we're (eventually) trying to make go away, but for complicated stuff like this it's probably best.